model_compression_toolkit/qunatizers_infrastructure/pytorch/base_pytorch_quantizer.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Dict, Any, List
+
+from model_compression_toolkit.core.common import Logger
+from model_compression_toolkit.core.common.constants import FOUND_TORCH
+from model_compression_toolkit.core.common.quantization.node_quantization_config import BaseNodeQuantizationConfig
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+
+from model_compression_toolkit.qunatizers_infrastructure.common.base_quantizer import BaseQuantizer, QuantizationTarget
+
+if FOUND_TORCH:
+
+    class BasePytorchQuantizer(BaseQuantizer):
+        def __init__(self,
+                     quantization_config: BaseNodeQuantizationConfig,
+                     quantization_target: QuantizationTarget,
+                     quantization_method: List[QuantizationMethod]):
+            """
+            This class is a base quantizer which validate the provide quantization config and define abstract function which any quantizer need to implment.
```

Comment:
"This class is a base **Pytorch** quantizer which validate**s** the provide**d** quantization config and define**s** **an** abstract function which any quantizer needs to impl**e**ment"

(I know that this is copied from the Keras base quantizer and that the mistakes appear there too, but I think it's better if we try not to duplicate them :) )

Reply:
This comment is relevant to all documentation, I won't add additional comments regarding documentation mistakes in this review